go.mod

```diff
@@ -10,11 +10,19 @@ replace (
 	sigs.k8s.io/controller-tools => github.com/christarazi/controller-tools v0.6.2
 )
 
+replace (
+	// due to https://github.com/vishvananda/netlink/issues/146
+	github.com/vishvananda/netlink => github.com/sayboras/netlink v1.2.0-beta.0.20220501043012-2057f84ba1e5
+	// due to https://github.com/vishvananda/netns/issues/23
+	github.com/vishvananda/netns => github.com/sayboras/netns v0.0.0-20220501043130-aa5e0960b51b
+)
```

Comment:
Out of curiosity: which packages in cilium/cilium fail to compile on windows due to these issues? Alternatively, we could try to make the respective packages in cilium/cilium compile on windows (i.e. add appropriate `//go:build` tags or rename files to `*_linux.go`) similar to the changes in https://github.com/cilium/cilium/pull/16806- Also see https://github.com/cilium/cilium/issues/16843

Reply:
this is good point, there are 3 packages as per below. I will go with this route as it's in our control.

```
github.com/cilium/cilium/pkg/option
github.com/cilium/cilium/pkg/mac
github.com/cilium/cilium/pkg/ip
```

Reply:
Thanks. I've added these packages to the list in https://github.com/cilium/cilium/issues/16843 as well.

Reply:
a small change for your review :) 

https://github.com/cilium/cilium/pull/19662